internal/docgen/store-definitions.go

```diff
@@ -30,33 +30,66 @@ func createUnifiedRuleFile(semgrepRuleFiles []SemgrepRuleFile) error {
 
 		scanner := bufio.NewScanner(inputFile)
 
-		var copying bool
+		// Skip until line with "rules:"
 		for scanner.Scan() {
-			line := scanner.Text()
-			if line == "rules:" {
-				copying = true
-				continue
+			if strings.HasPrefix(scanner.Text(), "rules:"){
```

Comment:
we saw that some rule files can have comments and additional content before the `rules:` line. will this change work for those cases?

I think the previous version worked

Reply:
yes, this for loop is exactly for that, so that we can skip anything before the "rules:" line

Reply:
ah, it's a different loop 🤦 
got it!